stanford-earth/matson
# EARTH-1462: Lazy load CSS background images

# READY FOR REVIEW

# Summary
- TL;DR - Set CSS background images to load when within the viewport

# Needed By (Date)
- Next push

# Steps to Test

1. Go /patterns
2. Verify that background images appear for:
     Link Banner, Photo Banner, Postcard, Section Expandable Banner, Section Header Banner, Section Highlight Banner, Section Quote Banner, and Section Spotlight Banner.
3. Inspect the page, go to the Network panel, and verify that the background images are not requested until they appear in the viewport.
4. Repeat steps #2 and #3 for the global footer.
3. Profit


# Associated Issues and/or People
- JIRA ticket EARTH-1462
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
